deploy/deploy_ancillaryRH7.yml

```diff
@@ -0,0 +1,142 @@
+- hosts: wdcreancil02
+  vars:  
+    custom_repo: true
+  vars_files:
+    - ~/RHACCOUNT
+  become: yes
+  become_method: sudo
+  pre_tasks:
+    - debug:
+        msg: "Variables: _uid = {{ _uid }}, _pwd = **********"
+#       msg: "Variables: _uid = {{ _uid }}, _pwd = {{ _pwd }}"
+    - name: register RHEL7.5
+      command: "/usr/bin/subscription-manager register --username {{ _uid }} --password {{ _pwd }} --auto-attach"
+      when: _uid != "" and _pwd != ""
+#     command: "/usr/bin/subscription-manager register --username <name> --password <password> --auto-attach"
+    - name: register RHEL7 repo
+      command: "/usr/bin/subscription-manager repos --enable=rhel-7-server-rpms --enable=rhel-7-server-extras-rpms --enable=rhel-7-server-optional-rpms"
+    - name: register RHEL7 RHSCL Repo
+      shell: "/usr/bin/subscription-manager repos --enable=rhel-server-rhscl-7-rpms"
+    - name: register RHEL7 DevTools
+      shell: "/usr/bin/subscription-manager repos --enable=rhel-7-server-devtools-rpms"
+    - name: list RHEL enabled repos
+      shell: "/usr/bin/subscription-manager repos --list-enabled"
+      register: enabled_repos
+    - debug:
+        msg: "RHEL Enabled Repos: {{ enabled_repos }}" 
+# tasks:
+    - name: download epel
+      shell:  wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -O /tmp/epel-release-latest-7.noarch.rpm   
+    - name: Install epel
+      shell:  yum -y localinstall /tmp/epel*rpm
+
+    - name: install X Window System
+      shell: yum -y groupinstall "X Window System"
+
+    - name: install XFCE
+      shell: yum -y groupinstall "xfce"
+
+    - name: install Evince
+      shell: yum -y install evince
+
+    - name: Install python-pip
+      shell: yum install -y python27 python-pip
+    - name: install gdown
+      shell: pip install gdown  
+#     yum:
+#       name: python27-python-pip
+#       state: latest
+    - name: Install python 3.6 and pip
+      shell: yum install -y python36 python36-pip
```

Comment:
Shouldn't we be calling the python3 role for this, not installing python separately?

Reply:
this is a specific directive for RHEL7, it could be placed in a role, but we probably would have to make more fine-grained than python3, probably will have to spell out python3.6, etc.